### PR TITLE
Add console support

### PR DESCRIPTION
**Pull request by Lord-Protector**

### CODE REVIEW

**Comment by Lord-Protector** [comment]
Sorry for two pull resquests. To lazy to clone second time :) Just used github online editor.

File: src/handlers/commandHandler.js [hunk 1]
```diff
@@ -215,13 +225,15 @@ export default class CommandHandler {
...
-    - \`/links <Battle Tag> <Region>\`: Provides links to websites providing more stats
+    - \`/stats <Battle Tag> <Region / Platform>\`: Retrieve player's stats
+    - \`/links <Battle Tag> <Region / Platform>\`: Provides links to websites providing more stats
 
     *Examples*
     US player: /stats LastBastion#12345
     EU player: /stats LastBastion#12345 eu
     KR player: /stats LastBastion#12345 kr
+    PS4 player: /stats LastBastion#12345 psn
+    Xbox One player: /stats LastBastion#12345 xbl
```
Comment (chesterhow) [inline src/handlers/commandHandler.js]: Could you remove the battle tag id? I just checked and it seems that console players' battle tags do not have the hash.
```
PS4 player: /stats LastBastion psn
Xbox One player: /stats LastBastion bbl
```

Comment (chesterhow) [inline src/handlers/commandHandler.js]: Here's an example https://playoverwatch.com/en-us/career/xbl/Meuq

**Comment by Lord-Protector** [comment]
Just send you two commits in patch-1 branch.
Also fix one strange bug:
I don't know why, but it works pretty good:
`if ((this.region == 'xbl') || (this.region == 'psn'))`
But this don't (by default it's just pick the fisrt (xbl)):
if (this.region === ('xbl' || 'psn'))
Now I fix it and on my instance all works nice.

File: src/handlers/commandHandler.js [hunk 2]
```diff
@@ -142,6 +143,10 @@ export default class CommandHandler {
       data = json.eu;
     } else if (this.region === 'kr') {
       data = json.kr;
+    } else if (this.region === 'psn') {
+      data = json.any;
+    } else if (this.region === 'xbl') {
```
Comment (chesterhow) [inline src/handlers/commandHandler.js]: Could you convert this to `else if ((this.region == 'xbl') || (this.region == 'psn'))` as well?

**Comment by Lord-Protector** [comment]
Push fix

File: src/handlers/commandHandler.js [hunk 3]
```diff
@@ -142,6 +143,8 @@ export default class CommandHandler {
       data = json.eu;
     } else if (this.region === 'kr') {
       data = json.kr;
+    } else if ((this.region == 'xbl') || (this.region == 'psn'))
```
Comment (chesterhow) [inline src/handlers/commandHandler.js]: Oh no you missed out an opening brace! I'm gonna do a quick fix on master

Comment (Lord-Protector) [inline src/handlers/commandHandler.js]: Sorry!

**Pull request merged**